### PR TITLE
devtodo: deprecate

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
license ref

> .\" todo is licensed under the GPL, version 2. A copy of the GPL should have been distributed with the source in the file COPYING

**Review by carlocab — COMMENTED** [review]
Why not just update it to the new version at https://swapoff.org/devtodo.html?

File: Formula/devtodo.rb [hunk 1]
```diff
@@ -75,6 +80,6 @@ def install
  #include <curses.h>
  #include <term.h>
 +#include <stdlib.h>
- 
+
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/devtodo.rb:83]: yeah, that was done by editor, let me revert it.

**Comment by mhmdanas** [comment]
@carlocab seems like v2 still does not support a lot of features from v1 ([source](https://github.com/alecthomas/devtodo2#devtodo1)). Given this, I think that updating the formula to v2 might be a disruptive change to its users.

**Comment by SMillerDev** [comment]
Well, upstream made that release so we are going to update to it.

File: Formula/devtodo.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class Devtodo < Formula
   homepage "https://swapoff.org/devtodo.html"
   url "https://swapoff.org/files/devtodo/devtodo-0.1.20.tar.gz"
```
Comment (SMillerDev) [inline Formula/devtodo.rb:4]:
```suggestion
  url "https://github.com/alecthomas/devtodo2/archive/refs/tags/2.1.tar.gz"
```

Comment (mhmdanas) [reply Formula/devtodo.rb:4]: Note that more changes to the formula are required to build v2, as it is written in Go, while v1 is written in C++.

**Pull request closed without merging**